Sources/armory/logicnode/ColorgradingGetHighlightNode.hx

```diff
@@ -6,24 +6,16 @@ class ColorgradingGetHighlightNode extends LogicNode {
 		super(tree);
 	}
 
-    override function get(from:Int):Dynamic {
-        if(from == 0) {
-            return armory.renderpath.Postprocess.colorgrading_highlight_uniforms[0][2];
```

Comment:
This was not a typo here (`colorgrading_highlight_uniforms` is a 2-dimensional array). Also this node has 6 outputs as before, not 5 as it is in this PR.

Reply:
Okay i will "unfix" this. Thanks!

Reply:
About this line: `armory.renderpath.Postprocess.colorgrading_highlight_uniforms[0][2];`

If i declare a variable like: `var colorgrading: String = armory.renderpath.Postprocess.colorgrading_highlight_uniforms`

It will work as this: colorgrading[0][2]

I think this is better for readability, but i don't have sure about the performance. Can you tell me if this works and if this takes some performance?

Reply:
Why `String`? Have a look at where it is defined:

https://github.com/armory3d/armory/blob/da1cf33930ce9a8b1865d35c128fe4842bef2933/Sources/armory/renderpath/Postprocess.hx#L36-L42


It's an array of arrays (2D), so instead of String you would need to use `Array<Array<Float>>` as the type. I'm not sure about the performance, it is a bit slower on one hand because it's saved in a variable (memory needs to be allocated and it has to be written into that variable), but on the other hand it's faster because the computer does not need to go from `armory` to `renderpath` and so on to read from that variable. I don't know,

Reply:
Hm, really interesting. 

I tough that if i put: `var ballName = ball.name.12.3.1.3.13`, the script would be interpreted like: he put `ballName` but i know that `ballName` is just a shorter name for `ball.name.123.1.3.13`. This is why i would put the variable as string.

Just a newbie :) Thanks for the explanation!

Reply:
What you describe does exist and it is called "inline variables". Haxe also has them if you explicitly declare them as `inline var`, but you can't use them inside of functions. But as I said, chances are good that Haxe automatically inlines this variable. If you're unsure, let it as it was without a variable.